examples/basic-preact/app/entry.client.tsx

```diff
@@ -0,0 +1,12 @@
+import { hydrate } from "react-dom";
+import { RemixBrowser } from "remix";
+
+const documentElement = document.documentElement;
+const apply = (n: HTMLElement) => document.replaceChild(n, documentElement);
+// Temp fix
```

Comment:
Preact currently fails when passing in `document` as that has a few nodes that aren't reconcilable 😅